crates/teloxide/src/update_listeners/webhooks/axum.rs

```diff
@@ -47,13 +47,13 @@ where
     <R as Requester>::DeleteWebhook: Send,
 {
     let Options { address, .. } = options;
+    let tcp_listener = tokio::net::TcpListener::bind(address).await.unwrap();
```

Comment:
this is not the right place to do it. this should be inside the Tokio::spawn and have the same error handling

Reply:
Done